support/crown/src/common.rs

```diff
@@ -332,6 +335,7 @@ pub fn implements_trait_with_env<'tcx>(
         kind: TypeVariableOriginKind::MiscVariable,
         span: DUMMY_SP,
     };
+    // in new nightlies: mk_substs -> mk_substs_from_iter
```

Comment:
I'm not sure what this comment means. Do you mind explaining it a bit?

Reply:
This note is obsolete. When we used old nightly version this comment suggested how to upgrade code for new nightly.